### PR TITLE
xinpianchang.StudioTrans version 1.2.4-117

### PR DESCRIPTION
**Pull request by oxygen-dioxide**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/23719)

### CODE REVIEW

**Review by vedantmgoyal9 — CHANGES_REQUESTED** [review]
Please change the locale. Its not English(US).

File: manifests/x/xinpianchang/StudioTrans/1.2.4-117/xinpianchang.StudioTrans.locale.zh-CN.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: xinpianchang.StudioTrans
+PackageVersion: 1.2.4-117
```
Comment (ItzLevvie) [inline manifests/x/xinpianchang/StudioTrans/1.2.4-117/xinpianchang.StudioTrans.locale.zh-CN.yaml:5]:
```suggestion
PackageVersion: 1.2.4
```

File: manifests/x/xinpianchang/StudioTrans/1.2.4-117/xinpianchang.StudioTrans.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: xinpianchang.StudioTrans
+PackageVersion: 1.2.4-117
```
Comment (ItzLevvie) [inline manifests/x/xinpianchang/StudioTrans/1.2.4-117/xinpianchang.StudioTrans.yaml:5]:
```suggestion
PackageVersion: 1.2.4
```

File: manifests/x/xinpianchang/StudioTrans/1.2.4-117/xinpianchang.StudioTrans.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: xinpianchang.StudioTrans
+PackageVersion: 1.2.4-117
```
Comment (ItzLevvie) [inline manifests/x/xinpianchang/StudioTrans/1.2.4-117/xinpianchang.StudioTrans.installer.yaml:5]:
```suggestion
PackageVersion: 1.2.4
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

**Please update the version folder as well so that it passes the pipelines.**

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**